x-pack/plugins/maps/public/classes/sources/wms_source/wms_layer_wizard.tsx

```diff
@@ -25,14 +24,14 @@ export const wmsLayerWizardConfig: LayerWizard = {
   }),
   icon: WebMapServiceLayerIcon,
   renderWizard: ({ previewLayers }: RenderWizardArguments) => {
-    const onSourceConfigChange = (sourceConfig: unknown) => {
+    const onSourceConfigChange = (sourceConfig: Partial<WMSSourceDescriptor>) => {
```

Comment:
Maybe consider using a `WMSSourceConfig` type where the arguments are required instead? See other comment below.